src/components/ProtectionRefInput.tsx

```diff
@@ -164,6 +166,7 @@ export default function ProtectionRefInput<
       } else if (id) {
         createRecord(id, selectedData as number[])
       }
+      reset(getValues())
```

Comment:
Hello @Taha - could you give me a quick description of what was broken, and how this fix works please?

Reply:
`isSubmitSuccessful` was not getting reset after first submission. So, I `reset` the form and set the values that already were there which triggered `isSubmitSuccessful` to turn to true from false.